force-app/main/default/classes/SOQL.cls

```diff
@@ -200,7 +221,8 @@ public with sharing class SOQL {
 	 * @return A list of Cases
 	 */
 	public static List<Case> getCasesWithStatus() {
-		return null; // Replace null with the variable you used to store the result
+		List<Case> casesWithStatus = [SELECT Status, CreatedDate, ID FROM Case WHERE Status = 'Open' ORDER BY CreatedDate LIMIT 10 OFFSET 5];
```

Comment:
Try to filter by isClosed instead of 'Open' since there are other statuses on case that can be considered open (ex. New, In progress)